examples/linearDynamicCantilever/onsasExample_linearDynamicCantilever.m

```diff
@@ -0,0 +1,223 @@
+%md# Dynamic Vibration modes of a Beam with fix nodes in both ends.
+%md---
+%md
+%mdIn this tutorial, the dynamic response of a beam with fix nodes in both ends is solved using ONSAS implementing the corotational model and the linear elastic model. The aim of this example is to validate the dynamic response of a beam, to do this the analytic solution is comapred with the ONSAS result and the error is estimated in order to validete the result. The Octave script of this example is available at [this url](https://github.com/ONSAS/ONSAS.m/blob/master/examples/uniformCurvatureCantilever/onsasExample_uniformCurvatureCantilever.m).
+%md
+%mdThe problem consists in a beam with fixed nodes in both ends. In a selected position of the beam a forced load with time dependency is applied $(F = F_o sin(wt))$, as it is shown in the figure.
+%md
+%md```@raw html
+%md<img src="assets/dynamicBeamHTML.svg" alt="structure diagram" width="500"/>
+%md```
+%md
+%mdBefore defining the structs, the workspace is cleaned, the ONSAS directory is added to the path and scalar geometry, material parameters and load values are defined.
+close all, clear all ;
+% add path
+addpath( genpath( [ pwd '/../../src'] ) );
+% External forced load parameters
+Fo     = 100; % N 
+w      = 2;    % rad/s
+% Time values
+tf     = 8;    % sec
+deltat = 0.1; % sec
+%md
+% Material scalar parameters
+E = 200e9 ;  nu = 0.3 ; rho = 700;
+% geometrical scalar parameters
+l = 10 ; ty = .3 ;  tz = .1 ;
+Iyy = ty*tz^3/12 ;
+Izz = tz*ty^3/12 ;
+% Number of elements of the mesh
+numElements = 50 ;
+%md
+%md## Analytic solution
+%md 
+%md The dynamic displacement of a forced beam describe by the next differential equation
+%md```math
+%md EI \frac{\partial^4 w}{\partial x^4} + \rho A \frac{\partial^2
+%w}{\partial t^2} = f(x,t)
+%md```math
+%md Implementig a solution w(x,t) = W(x)T(t) it is possible to find:
+%md```math 
+%md w(x,t) = \frac{2fo}{\rho A l}\sum_{n=1}^{\infty } \frac{1}{w_{n}^2 - w^2}\sin(\frac{n \pi a}{l})\sin(\frac{n \pi x}{l})\sin(wt)
+%md```math
+%md 
+%md## External Load application node
+a  = l*(numElements/2+1)/numElements; 
+%md
+%md## Analytic solution of a beam with fix nodes in both ends. 
+analyticSol  = @(x, t, wn, n) (1/(wn^2 - w^2))*sin(n*pi*a/l)*sin(n*pi*x/l)*sin(w*t);
+%md
+t  = 0:deltat:tf; % time vector
+x  = 0:l/numElements:l; % beam mesh
+analyticDisV  = zeros(numElements+1, length(t));
+analyticDisW  = zeros(numElements+1, length(t));
+for i = 1:1000
+    wnY(i) = ((i*pi)^2)*sqrt(E*Izz/rho/(ty*tz)/(l^4));
+    wnW(i) = ((i*pi)^2)*sqrt(E*Iyy/rho/(ty*tz)/(l^4));
+    for j = 1:numElements+1
+        for k = 1:length(t)
```

Comment:
si no me equivoco esto está corriendo unas 1000 x 50 x 80 veces... 4 millones de veces... para hacer un cálculo auxiliar de la solución analítica... sugiero vectorizar esto. https://octave.org/doc/v7.1.0/Basic-Vectorization.html

Reply:
Veo q esto quedó resuelto

Reply:
Jorge, en el ultimo pull request que mande reduje las operaciones a operaciones vectoriales en cada modo. De esta forma se realiza un for en la cantidad de modos. Dado que avanzar en los modos una de las entradas del vector tienda a cero se puede reducir el for a no más de 10 modos reduciendo las operaciones a realizar. Ee esta forma la cantidad de operaciones vectoriales se reduce y se optimiza el proceso. Intenta llevar todo a vectores pero no encontré la forma, lo podemos revisar para seguir aprendiendo de estas operaciones de mi parte.

Reply:
Dale genial. Si. Vectorizar siempre ayuda...ya sea en Matlab u Octave. Por ahora veo q el ci toma un tiempo razonable asi que por ahora está bien.